### ISSUE TITLE
No effect if set the bearing of FollowPuckViewportStateOptions to null

### ISSUE THREAD

**Issue by ystsoi** [mapbox/mapbox-maps-android#2110]
## Environment

- Maps SDK Version: 10.14.0

## Observed behavior and steps to reproduce

The map does not follow the location, using the following code:
```
viewportPlugin.transitionTo(
    viewportPlugin.makeFollowPuckViewportState(
        FollowPuckViewportStateOptions.Builder()
            .bearing(null)
            .build()
    )
)
```

## Expected behavior

The map should at least follow the location, similar to:
```
viewportPlugin.transitionTo(
    viewportPlugin.makeFollowPuckViewportState(
        FollowPuckViewportStateOptions.Builder()
            .build()
    )
)
```
or:
```
viewportPlugin.transitionTo(
    viewportPlugin.makeFollowPuckViewportState(
        FollowPuckViewportStateOptions.Builder()
            .bearing(FollowPuckViewportStateBearing.Constant(0.0))
            .build()
    )
)
```

**Comment by SamYStudiO** [comment]
Same here

**Comment by jush** [comment]
Hi thanks for reporting this issue. We'll look into it.

Meanwhile, you can workaround it by setting bearing to the default value `FollowPuckViewportStateBearing.SyncWithLocationPuck` instead of `null`:
```Kotlin
viewportPlugin.transitionTo(
    viewportPlugin.makeFollowPuckViewportState(
        FollowPuckViewportStateOptions.Builder()
            .bearing(FollowPuckViewportStateBearing.SyncWithLocationPuck)
            .build()
    )
)
```

**Comment by kiryldz** [comment]
Closing as the issue was fixed in v11.0.0 if I get it right.
